modules/source/gemfire/config/gemfire.groovy

```diff
@@ -0,0 +1,19 @@
+beans {
+	xmlns([gfe   : 'http://www.springframework.org/schema/gemfire',
+		   intgfe: 'http://www.springframework.org/schema/integration/gemfire',
+		   si    : 'http://www.springframework.org/schema/integration'])
+
+	def xdhome = environment.getProperty('xd.home')
+	importBeans "file:${xdhome}/modules/common/gemfire-source.groovy"
+
+	gfe.'client-region'(id: 'region', 'cache-ref': 'client-cache', name: '${regionName}', 'data-policy': 'EMPTY') {
+		gfe.'key-interest' {
+			bean(id: 'key', 'class': 'java.lang.String') {
+				'constructor-arg'(value: 'ALL_KEYS')
+			}
+		}
+	}
+
+	si.channel(id: 'output')
```

Comment:
not sure we need to redeclare the output channel since it is already present in the gemfire-source